Packs/Expanse/Integrations/Expanse/Expanse.py

```diff
@@ -914,6 +957,116 @@ def exposures_command():
     return_outputs(human_readable, ec, exposures)
 
 
+def domains_for_certificate_command():
+    """
+
+    :return:
+    """
+    search = demisto.args()['common_name']
+    params = {
+        "commonNameSearch": search
+    }
+    token = do_auth()
+
+    observed_ips = []
+    matching_domains = []
+
+    certificates = _fetch_certificates(params=params, token=token)
+    for certificate in certificates:
+        certificate_details = _fetch_certificate(md5_hash=certificate['certificate']['md5Hash'], token=token)
+        for ip in certificate_details['details']['recentIps']:
+            observed_ips.append(ip['ip'])
+
+    for ip in observed_ips:
```

Comment:
i dont think we need that loop
we are already iterating all objects above when building `observed_ips` list, why not just do it there?

Reply:
good catch